### PR TITLE
Reg login visual improvements

### PR DESCRIPTION
**Pull request by IstrajI**

### CODE REVIEW

File: app/src/main/java/pro/itshark/moneysplitter/presentation/regLogin/login/LoginViewModel.kt [hunk 1]
```diff
@@ -11,10 +12,13 @@ import javax.inject.Inject
 class LoginViewModel
 @Inject constructor(private val userUseCases: UserUseCases): ViewModel() {
     val stateLiveData = MutableLiveData<LoginScreenState>()
-
     val credits = LoginModel()
+    val requestFailed: ObservableField<Boolean> = ObservableField(false)
```
Comment (BlaizMiko) [inline app/src/main/java/pro/itshark/moneysplitter/presentation/regLogin/login/LoginViewModel.kt]: `isRequestSuccess` - cause its boolean and in most cases we must to use some positive way for naming

Comment (IstrajI) [reply app/src/main/java/pro/itshark/moneysplitter/presentation/regLogin/login/LoginViewModel.kt]: Renamed

File: app/src/main/java/pro/itshark/moneysplitter/presentation/regLogin/registration/RegistrationViewModel.kt [hunk 2]
```diff
@@ -27,12 +27,12 @@ class RegistrationViewModel
...
 
     private fun onRegistrationSuccess(userEntry: UserEntry) {
-        requestFailed.set(false)
+        isRequestSuccess.set(false)
         stateLiveData.value = RegistrationSuccessState()
     }
 
     private fun onError(error: Throwable) {
-        requestFailed.set(true)
+        isRequestSuccess.set(true)
```
Comment (BlaizMiko) [inline app/src/main/java/pro/itshark/moneysplitter/presentation/regLogin/registration/RegistrationViewModel.kt]: U set true for `request success` variable inside onError

Comment (IstrajI) [reply app/src/main/java/pro/itshark/moneysplitter/presentation/regLogin/registration/RegistrationViewModel.kt]: Fck

**Pull request merged**